main.go

```diff
@@ -94,3 +124,29 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// newLogger returns a logger configured for dev or production use.
```

Comment:
Seems like this is generic enough to go in pkg (in the future some time ..)

Reply:
Maybe, I'm not convinced that this will not break helm-controller due to go-logr -> controller-runtime -> client-go <- helm

Reply:
What is go-logr doing depending on controller-runtime?? Oh well